js/main.js

```diff
@@ -110,6 +111,116 @@ var Menu = {
   },
 };
 
+// AJAX
+Ajaxy = {
+  init: function() {
+    var _this = this;
+    
+    var siteURL = "http://" + top.location.host.toString();
+
+    _this.$ajaxyLinks = $("a[href^='" + siteURL + "'], a[href^='/'], a[href^='./'], a[href^='../'], a[href^='#']");
+    //_this.$elementsToHide = $('.nav, #main-container, #three-scene');
+
+    // Find all ajaxy links and bind ajax event
+    _this.$ajaxyLinks.click(function(event) {
+      event.preventDefault();
+
+      var url = event.currentTarget.href;
+
+      _this.ajaxLoad(url);
+
+    });
+
+    $(document).ready( function() {
```

Comment:
there is already a doc ready. Does this just bind another function to that event?